install-gcloud.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Install dependencies
+sudo apt-get install apt-transport-https ca-certificates gnupg -y
+
+# Add the GCP package repository
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+
+# Add the GCP public key to the system's list of trusted keys
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
+# Update package list and install the Cloud SDK
+sudo apt-get update && sudo apt-get install google-cloud-cli -y
+
+echo "Google Cloud SDK installed successfully!"
+
```

Comment:
**Question:** For some reason, gcloud is not installing for me, is it the same for you?

```

Fetched 378 kB in 2s (209 kB/s)   
Reading package lists... Done
E: Could not get lock /var/lib/dpkg/lock-frontend. It is held by process 5844 (apt-get)
N: Be aware that removing the lock file is not a solution and may break your system.
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?
Google Cloud SDK installed successfully!
gitpod /workspace/open-cloud-dev-box (main) $ gcloud

```

Reply:
Yeah, so if I manually run the script myself, this now works. I think there's some apt collision on installation. A fix could be to add a `gp sync-await <some-well-named-thing>` then `gp sync-done <some-well-named-thing>`

https://www.gitpod.io/docs/references/gitpod-cli#sync-await
https://www.gitpod.io/docs/references/gitpod-cli#sync-done

Reply:
had it as well, this ensures that the first apt-get runs without interruption